### PR TITLE
✨ Convert unusable containers to mp4

### PR DESCRIPTION
**Pull request by pajowu**

### CODE REVIEW

File: app/src/core/ffmpeg.ts [hunk 1]
```diff
@@ -278,6 +278,20 @@ export async function convertToWav(
   return fileData;
 }
 
+export async function copyToMp4(input_path: string): Promise<Buffer> {
+  const tempdir: string = await getTempDir();
+  const outputFile = path.join(tempdir, 'output.mp4');
+
+  const cmd = new FFmpegCommand({ y: undefined });
```
Comment (anuejn) [inline app/src/core/ffmpeg.ts:285]: what does this y: undefined do?

Comment (pajowu) [reply app/src/core/ffmpeg.ts:285]: I don't know. I copied that from convertToWav

Comment (pajowu) [reply app/src/core/ffmpeg.ts:285]: Ahhhh, I see where that comes from. I think we also copied it to convertToWav from the ffmpeg exporter. This leads to fessonia settting the `-y` command line flag which allows ffmpeg to overwrite the output file if it already exists

Comment (pajowu) [reply app/src/core/ffmpeg.ts:285]: I don't care about this flag, should we remove it or keep it?

Comment (anuejn) [reply app/src/core/ffmpeg.ts:285]: Ah it is fine. I thought that undefined might be the wrong value (expected true here) but this is fine :)

**Pull request merged**